app/controllers/event_attendances_controller.rb

```diff
@@ -0,0 +1,50 @@
+class EventAttendancesController < ApplicationController
+  layout "site"
+  before_action :authorize_web
+  before_action :set_event_attendance, :only => [:update]
+
+  authorize_resource
+
+  def create
+    attendance = EventAttendance.new(attendance_params)
+    attendance.intention = intention
+    if attendance.save!
+      redirect_to event_path(attendance.event), :notice => "Attendance was successfully saved."
```

Comment:
All these strings need to be through the i18n system. I find it best to do so right from the start, since it's time consuming to go back through all the controllers and views later on.

Reply:
yup yup